activerecord/lib/active_record/database_configurations/hash_config.rb

```diff
@@ -103,6 +103,14 @@ def adapter
       def schema_cache_path
         configuration_hash[:schema_cache_path]
       end
+
+      def database_tasks
```

Comment:
I think this would give a better idea of the method's purpose:

```suggestion
      def database_tasks?
```

Reply:
Also I think this should match the other tasks and use `fetch` and include `replica` since those dont run tasks either: Ex: `configuration_hash.fetch(:database_tasks, true) || replica?`

Reply:
Ok I have made it `database_tasks?` and changed the method to use replica and fetch. Doesnt look exactly as you described it though, but I think its correct.